### ISSUE TITLE
Data augmentation

### ISSUE THREAD

**Issue by jrussell25** [Hekstra-Lab/yeast-mrcnn#3]
Would be good to use albumentations for some real data augmentation. Torchvision has limited support for image -> boxes and masks which seemed not even worth bothering with but alblumentations has several.

Main obstacle here is the albumentations uses a different organization scheme for the training data than is natural to use with torchvision models so would need to rewrite the datasets to go from one to the other. 

https://albumentations.ai/docs/getting_started/simultaneous_augmentation/
